generators/builtins/common.kt

```diff
@@ -61,7 +61,7 @@ fun progressionIncrementType(kind: ProgressionKind) = when (kind) {
 
 fun areEqualNumbers(v: String) = "$v == other.$v"
 
-fun hashLong(v: String) = "($v xor ($v ushr 32))"
+fun hashLong(v: String) = "$v.hashCode()"
```

Comment:
There's not much sense in having this function now, so it can be inlined.

Reply:
I don't really understand how you would inline it, can you be more precise?

Reply:
By "inline" I mean substitute function calls with its body and remove the function. There's a refactoring in IDEA for that.